License

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```

Comment:
I think you should include your full name and year in this part instead of the default place holders